src/print.c

```diff
@@ -35,11 +38,18 @@ struct pg_print_config {
 struct pg_print_state {
 	struct pg_brick brick;
 	FILE *output;
+	pcap_dumper_t *dumper;
 	uint16_t *type_filter;
 	int flags;
 	struct timeval start_date;
 };
 
+#define PCAP_SNAPSHOT_LEN 65535
+static char print_data[PCAP_SNAPSHOT_LEN];
+static uint64_t start_cycles;
+static struct timeval start_time;
+static uint64_t hz;
+
```

Comment:
You cannot use those globals for all print bricks.
- print_data can be used at the same time by two different print bricks
- start_cycles, start_time and hz are also set during brick init or during burst

You should move all this in brick's state